api/app/clients/chatgpt-client.js

```diff
@@ -7,6 +7,7 @@ const askClient = async ({
   parentMessageId,
   conversationId,
   model,
+  apiKey,
```

Comment:
now i understand it as being the browser token, so perhaps changing that from apiKey to browserToken will be better